web/src/state_data.ts

```diff
@@ -42,6 +42,7 @@ export const realm_schema = z.object({
             id: z.number(),
             name: z.string(),
             order: z.number(),
+            required: z.optional(z.boolean()),
```

Comment:
Why is this option? It seems like it should just be a `boolean`.

Reply:
Fixed in the recent commit.